deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -2,9 +2,19 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.namespace }}
+  labels: 
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: restricted
```

Comment:
```suggestion
```
If we enforce: restricted then we do not need to warn it.
The containers that cannot run as restricted will not be admitted so has no point to warn

Reply:
@anik120 ^